client_tipc.c

```diff
@@ -1067,6 +1073,119 @@ static struct storage_client_session *chan_context_to_client_session(struct ipc_
 	return session;
 }
 
+static rpmb_block_t *get_rpmb_block(void)
```

Comment:
get_rpmb_block() 
can you use a better name? as it will confuse people. e.g. which block do you want to get?